src/cli/record/import.ts

```diff
@@ -75,7 +75,7 @@ const builder = (args: yargs.Argv) =>
       requiresArg: true,
     })
     .option("file-path", {
-      describe: 'The path to source file.\nThe file extension should be ".csv"',
+      describe: 'The path to source file. The file extension should be ".csv"',
```

Comment:
I think you can try the custom completion function.
https://yargs.js.org/docs/#api-reference-completioncmd-description-fn

The `.completion()` accepts a custom function before outputting the completion script.
You can escape line breaks inside it without removing the line break from the description.

Reply:
(Possibly, you can resolve the alias problem with this too.)